internal/input/bucketlist_entries.go

```diff
@@ -60,3 +62,33 @@ func readBucketList(archive historyarchive.ArchiveInterface, checkpointSeq uint3
 
 	return entrySlice, nil
 }
+
+func GetLedgerCloseMeta(end uint32, env utils.EnvironmentDetails) (xdr.LedgerCloseMeta, error) {
+	ctx := context.Background()
+	captiveCoreToml, err := ledgerbackend.NewCaptiveCoreTomlFromFile(
+		env.CoreConfig,
+		ledgerbackend.CaptiveCoreTomlParams{
+			NetworkPassphrase:  env.NetworkPassphrase,
+			HistoryArchiveURLs: env.ArchiveURLs,
+			Strict:             true,
+		},
+	)
+	if err != nil {
+		return xdr.LedgerCloseMeta{}, err
+	}
+	backend, err := ledgerbackend.NewCaptive(
+		ledgerbackend.CaptiveCoreConfig{
+			BinaryPath:         env.BinaryPath,
+			Toml:               captiveCoreToml,
+			NetworkPassphrase:  env.NetworkPassphrase,
+			HistoryArchiveURLs: env.ArchiveURLs,
+		},
+	)
+
+	ledgerCloseMeta, err := backend.GetLedger(ctx, end)
+	if err != nil {
+		return xdr.LedgerCloseMeta{}, errors.Wrap(err, "error getting ledger from the backend")
+	}
+
+	return ledgerCloseMeta, nil
+}
```

Comment:
We don't need to address now, but I think it would be better for us to write a new interface in `utils/main.go` that will instantiate a new CaptiveCore and just use the GetLedger method for retrieving the `LedgerCloseMeta`. This current design is clunky and we do not have a consistent captive core interface.